### PR TITLE
[JSInterop] Skip test until we move to the new .NET Core JSON implementation.

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs [hunk 1]
```diff
@@ -2184,12 +2214,18 @@ public bool Remove(KeyValuePair<TKey, TValue> item)
...
+                        return _dictionary.GetEnumerator();
+                    }
                 }
 
                 System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
                 {
-                    return _dictionary.GetEnumerator();
+                    lock (_lock)
+                    {
+                        return _dictionary.GetEnumerator();
```
Comment (pakrym) [inline src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs]: This won't help, enumerator is being used outside the lock.

Comment (javiercn) [reply src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs]: Any cheap way of fixing this? I can create an enumerator wrapper and have it release the lock on Dispose but anything else seems overkill.

I'm not concerned too much about this as the callstacks that we have show another method is the one causing issues, but if there is something trivial (15 min) I'll do it.

Comment (rynowak) [reply src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs]: You can have it copy the underlying data. 

Is it feasible to eliminate this from the code? - ie does anything call this? If this is internal to simplejson can the calling code take the lock

File: src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs [hunk 2]
```diff
@@ -2117,12 +2117,21 @@ public void Add(TKey key, TValue value)
...
                 }
 
                 public ICollection<TKey> Keys
                 {
-                    get { return _dictionary.Keys; }
+                    get
+                    {
+                        lock(_lock)
+                        {
+                            return _dictionary.Keys;
```
Comment (rynowak) [inline src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs]: This is also broken.

File: src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs [hunk 3]
```diff
@@ -2132,18 +2141,33 @@ public bool Remove(TKey key)
...
                 }
 
                 public ICollection<TValue> Values
                 {
-                    get { return _dictionary.Values; }
+                    get
+                    {
+                        lock(_lock)
+                        {
+                            return _dictionary.Values;
```
Comment (rynowak) [inline src/JSInterop/Microsoft.JSInterop/src/Json/SimpleJson/SimpleJson.cs]: broken

**Pull request merged**